src/components/header/header.tsx

```diff
@@ -91,7 +96,9 @@ export class Header {
                 </div>
             </div>,
             <div class="actions">
-                <slot />
+                <slot name="actions">
+                    <slot />
+                </slot>
             </div>,
```

Comment:
````suggestion
            <slot name="actions" class="actions">
                    <slot />
            </slot>,
````

Reply:
isn't this cleaner?

With the above, we would need this in `header.scss`
```scss
slot[name='actions'] {
```

instead of this 👇 
```scss
.actions {
```

in two places

Reply:
If we don't need the div for styling, I'm only happy to get rid of it 👍

Reply:
will you fixup! so that I can approve

Reply:
Sure, I'll fix! 👍

Reply:
Hmm… 🤔 This doesn't seem to work 😞

<img width="990" alt="A screenshot from VSCode, showing an error: Property 'class' does not exist on type 'SlotAttributes'" src="https://user-images.githubusercontent.com/154725/213199939-1257e357-d3d4-4c1b-b3b7-a3391c17be36.png">

I didn't get an error when running `npm run build`, but when running `npm start`, I _do_ get an error, and it prevents the docs app from working when served locally, so that's a show-stopper.

Reply:
<img width="935" alt="A screenshot showing that running `npm start` fails with the same error as in the screenshot above" src="https://user-images.githubusercontent.com/154725/213201500-da7ba658-4bc1-432f-b978-7250eb387642.png">

Reply:
I think we need to keep it the way it is in this PR… 😞

Reply:
@Kiarokh Approved?

Reply:
No worries. I think you missed that my suggestion didn't have the `class=`. I pushed a fixup to show what I mean

Reply:
I also couldn't control myself, and updated the basic example to be more simple and easier to grasp. So I pushed an additional `docs` commit as well. What do you think?

Reply:
> I think you missed that my suggestion didn't have the `class=`.

Ah! Yes, or no, I did see that, but then thought "if we keep the class, we don't have to make those changes in the css you mentioned", but then when putting a `class` attribute on the slot didn't work I had completely forgotten about the fact that I had added that myself. I feel so stupid sometimes… 🤦‍♂️🤣

Reply:
> I also couldn't control myself, and updated the basic example to be more simple and easier to grasp. So I pushed an additional `docs` commit as well. What do you think?

Oh, yes, so much better! Thank you! 👍

One thing I thought of now though: That first example is pretty weird. The information presented in that header looks like it would belong in a snackbar instead. We don't have to fix that here and now, but I think we could make that example a lot less misleading 😅

Reply:
totally agree!